rfc/6/index.md

```diff
@@ -0,0 +1,190 @@
+# RFC-6: Multiscale
+
+Turn the `multiscales` array into a single `multiscale` object.
+
+
+## Status
+
+This proposal is very early. Status: D1
+
+```{list-table} Record
+:widths: 8, 20, 20, 20, 15, 10
+:header-rows: 1
+:stub-columns: 1
+
+*   - Role
+    - Name
+    - GitHub Handle
+    - Institution
+    - Date
+    - Status
+*   - Author
+    - Norman Rzepka
+    - @normanrz
+    - scalable minds
+    - 2024-12-03
+    - 
+*   - Author
+    - David Stansby
+    - @dstansby
+    - University College London
+    - 2024-12-03
+    - 
+*   - Endorser
+    - Davis Bennett
+    - @d-v-b
+    - 
+    - 2024-12-12
+    -
+*   - Endorser
+    - Will Moore
+    - @will-moore
+    - OME, University of Dundee
+    - 2024-12-12
+    -
+*   - Endorser
+    - Lachlan Deakin
+    - @LDeakin
+    - Australian National University
+    - 2024-12-17
+    -
+*   - Endorser
+    - Joel Lüthi
+    - @jluethi
+    - BioVisionCenter, University of Zurich
+    - 2024-12-18
+    -
+*   - Endorser
+    - Eric Perlman
+    - @perlman
+    - 
+    - 2024-12-18
+    -
+```
+
+## Overview
+
+This RFC proposes to change the `multiscales` array into a single `multiscale` object.
+
+## Background
+
+The current spec of OME-Zarr (version 0.5) defines that the metadata for a multiscale is stored in a `multiscales` array.
+
+However, there seem to only very few OME-Zarr images with mutltiple multiscales in the wild. There is one example from IDR: [4995115.zarr](https://ome.github.io/ome-ngff-validator/?source=https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0050A/4995115.zarr)
+
+Additionally, most visualization and processing tools today simply process the first multiscale object in the `multiscales` array, ignoring any subsequent entries. Here is a selection of tools that only load the first multiscale object:
+
+- [neuroglancer](https://github.com/google/neuroglancer/blob/master/src/datasource/zarr/ome.ts#L265-L310)
+- [vizarr](https://github.com/hms-dbmi/vizarr/blob/main/src/utils.ts#L88)
+- [itk-vtk](https://github.com/Kitware/itk-vtk-viewer/blob/master/src/IO/ZarrMultiscaleSpatialImage.js#L173)
+- [OMERO](https://github.com/ome/ZarrReader/issues/44)
+
+The current spec seems to acknowledge that this is to be expected to some degree in the following excerpt:
+
+> If only one multiscale is provided, use it. Otherwise, the user can choose by name, using the first multiscale as a fallback:
+> 
+> ```python
+> datasets = []
+> for named in multiscales:
+>     if named["name"] == "3D":
+>         datasets = [x["path"] for x in named["datasets"]]
+>         break
+> if not datasets:
+>     # Use the first by default. Or perhaps choose based on chunk size.
+>     datasets = [x["path"] for x in multiscales[0]["datasets"]]
+> ```
+
+
+This RFC aims to codify what already seems to be common practice by moving from a multiscales array to a single multisclaes object. This will reduce complexity for implementations.
```

Comment:
```suggestion
This RFC aims to codify what already seems to be common practice by moving from a multiscales array to a single multiscale object. This will reduce complexity for implementations.
```